src/components/SaveBar/SaveBar.module.scss

```
@@ -0,0 +1,23 @@
+@use '../../styles/global';
+
+.savBarWrapper {
```

se o nome do componente é savebar, troca a classe pro nome correto
```suggestion
.saveBarWrapper {
```